whitepapers/Tables.md

```diff
@@ -2,4 +2,39 @@
 
 
 
+## Pipeline configuration
 
+### Source
+A Source is a place metadata comes from.
+
+#### Columns
+* `name` -- Short, unique string
+* `long_title` -- full, human-friendly name
+* `home_page` -- URL
+* `favicon` -- 
+* `user_id` -- PK of the user with permission to submit data as this source (TODO: replace with django permissions stuff)
+
+### Harvester
+
+#### Columns
+* `key` -- Unique key that can be used to get the corresponding Harvester subclass
+* `version` --
```

Comment:
Yeah, it makes more sense to keep version only on the class, so they can't get out of sync. It felt weird having a one-column table, but I guess there's nothing wrong with that.